### PR TITLE
web: Add Hilbert map

### PR DESCRIPTION
**Pull request by jakubjancicka**
Add page with visualization of IPv4 address space on NERD web. Visualization uses Hilbert curve to display IPv4 address space. Pixel on visualization represents a certain network and its value is sum of reputation scores of IP addresses in given network. 

It uses iframe for displaying visualization. It is probably a temporary solution but it is the quickest one due to the remaining time.

### CODE REVIEW

**Comment by jakubjancicka** [comment]
@vaclavbartos

**Comment by vaclavbartos** [comment]
Thanks for the pull request, I'll test it and if it works well, merge it.

File: NERDweb/nerd_main.py [hunk 1]
```diff
@@ -1242,6 +1242,10 @@ def iplist():
         return Response('ERROR: Database connection error', 503, mimetype='text/plain')
 
 
+# ******************** Map ********************
+@app.route('/map/')
+def map():
```
Comment (vaclavbartos) [inline NERDweb/nerd_main.py:1247]: Here you overwrite the builtin `map` function, which breaks everything. Rename it.

File: NERDweb/templates/layout.html [hunk 2]
```diff
@@ -135,7 +135,7 @@
   {%- if config.logo_link %}</a>{% endif -%}
  </div>
  <div id="navigation">
-  <a href="{{ url_for('ips') }}">IP search</a> | <a href="{{ url_for('ip') }}">IP detail</a> | <a href="{{ url_for('data_index') }}">Data</a>
+  <a href="{{ url_for('ips') }}">IP search</a> | <a href="{{ url_for('ip') }}">IP detail</a> | <a href="{{ url_for('data_index') }}">Data</a> | <a href="{{ url_for('map') }}">Map</a>
```
Comment (vaclavbartos) [inline NERDweb/templates/layout.html:138]: Don't forget to use the new name here.

**Pull request merged**